test/lib/ldap_test_helper.rb

```diff
@@ -85,8 +85,8 @@ def ad_group_dn(name = 'group')
     "cn=#{name},#{@config.group_base}"
   end
 
-  def ad_user_payload
-    [{ :memberof => [ad_group_dn] }]
+  def ad_user_payload(name)
+    [{ :memberof => [ad_group_dn], :distinguishedname => [ad_user_dn(name)] }]
   end
```

Comment:
Can you please make the `name` argument optional and keep at least one old test that makes the query without `distinguishedname`? I want to make sure the code works for old code too.